docs/source/getting_started.rst

```diff
@@ -0,0 +1,21 @@
+Getting Started
+===============
+
+The Data Resource Generator (DRG) is a database and API as configuration application.
+
+Terms
+-----
+
+Data Resource
```

Comment:
Can you put the term in **bold font** and add a colon? Then, the reader more easily knows where-to-look.

Reply:
RST automatically handles term definition. I think it bolds and italicizes them -- https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists